package.json

```diff
@@ -1,10 +1,13 @@
 {
   "devDependencies": {
-    "vitepress": "^1.0.0-beta.3"
+    "@mermaid-js/mermaid-mindmap": "^9.3.0",
+    "mermaid": "9.1.0",
```

Comment:
So it works fine with an older mermaidjs version?

Reply:
The newest version broke the plugin. The author is working on a fix. Meanwhile, using the older version is the workaround.